Epoch [1/100]: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 26/26 [00:00<00:00, 127.13it/s, loss=0.303]
Clipping input data to the valid range for imshow with RGB data ([0..1] for floats or [0..255] for integers). Got range [0.0..4.999935].
Clipping input data to the valid range for imshow with RGB data ([0..1] for floats or [0..255] for integers). Got range [-1.4168841..1.1329886].
Traceback (most recent call last):
  File "/mnt/hdd/pengy/insole_anno/train.py", line 100, in <module>
    train(model, train_loader, test_loader, criterion, optimizer, num_epochs=num_epochs, device="cuda:0" if torch.cuda.is_available() else "cpu")
  File "/mnt/hdd/pengy/insole_anno/train.py", line 33, in train
    log_results(y, outputs, epoch)
  File "/mnt/hdd/pengy/insole_anno/train.py", line 76, in log_results
    plt.savefig(f"results/{epoch}.png")
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/pyplot.py", line 1223, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/figure.py", line 3388, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2218, in print_figure
    result = print_method(
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2068, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py", line 496, in print_png
    self._print_pil(filename_or_obj, "png", pil_kwargs, metadata)
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py", line 445, in _print_pil
    mpl.image.imsave(
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/matplotlib/image.py", line 1676, in imsave
    image.save(fname, **pil_kwargs)
  File "/home/pengy/.conda/envs/torch/lib/python3.10/site-packages/PIL/Image.py", line 2456, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/hdd/pengy/insole_anno/results/0.png'